src/app/shared/pipes/default-null-or-empty/default-null-or-empty.pipe.spec.ts

```diff
@@ -13,7 +13,7 @@ describe('DefaultNullOrEmpty pipe tests', () => {
   });
 
   it('transforms "     " to "-"', () => {
-    expect(pipe.transform('     ')).toBe('-');
+    expect(pipe.transform('')).toBe('-');
```

Comment:
Please revert this change - it un-does a fix applied to `develop`

Reply:
will use back ticks to bypass the formatter.